plonky2x/core/src/frontend/vars/bytes32.rs

```diff
@@ -110,6 +110,20 @@ impl SSZVariable for Bytes32Variable {
     }
 }
 
+impl<L: PlonkParameters<D>, const D: usize> CircuitBuilder<L, D> {
+    // Returns a Bytes32Variable with the first `num_bits` set to 0.
+    pub fn zero_top_bits(&mut self, original: Bytes32Variable, num_bits: usize) -> Bytes32Variable {
```

Comment:
Just a remark but we should probably say somewhere that Bytes32Variable is big endian.

Reply:
nit: instead of top, maybe you use the word `be`

Reply:
Yes, alternatively, rename the function to `mask_be_bits`